b2/raw_api.py

```diff
@@ -454,11 +454,13 @@ def check_b2_filename(self, filename):
 
         :param filename: A proposed filename in utf-8.
```

Comment:
I should have noticed this before...

The name passed into `upload()` is unicode, not utf-8.  And that's what's passed in here.

Reply:
Fixed.